src/Tasks/AlgoliaIndexTask.php

```diff
@@ -222,7 +228,12 @@ private function syncCreatedPagesWithPageAlgoliaObjectIDHolders($pages) {
     }
 
     /**
-     * SynChanges removes old pages, updates pages and adds new pages to Algolia.
+     * Sync only pages with changes since the last sync, removed pages and added pages
+     *
+     * When a sync has been done we have a reference point and now can sync only the changes
+     * Their have been holders created containing ID's of the pages that have been deleted, we can remove those from algolia
+     * Their is a last sync date and we can get the pages that have been changed since that day, update those in algolia
+     * All the pages not having a holder are not synced and those are the pages being added since the last sync, add those to algolia
```

Comment:
All the pages **that do not have** a holder are not synced. Those are the pages being added since the last sync. Add those to Algolia.